tests/e2e/performance/test_pvc_creation_deletion_performance.py

```diff
@@ -0,0 +1,291 @@
+            f"PVCs is {creation_st_deviation_percent}%."
+        )
+
+        deletion_st_deviation = statistics.stdev(deletion_time_measures)
+        deletion_st_deviation_percent = deletion_st_deviation / deletion_average * 100.0
+        if deletion_st_deviation_percent > accepted_deletion_deviation_percent:
+            raise ex.PerformanceException(
+                f"{msg_prefix} PVC deletion time deviation is {deletion_st_deviation_percent}%"
+                f"and is greater than the allowed {accepted_deletion_deviation_percent}%."
+            )
+
+        log.info(
+            f"{msg_prefix} The standard deviation percent for deletion of {len(deletion_time_measures)} sampled PVCs "
+            f"is {deletion_st_deviation_percent}%."
+        )
+
+        # all the results are OK, the test passes, push the results to the codespeed
+        push_to_pvc_time_dashboard(self.interface, "1-pvc-creation", creation_average)
+        push_to_pvc_time_dashboard(self.interface, "1-pvc-deletion", deletion_average)
+
+    def write_file_on_pvc(self, pvc_obj, filesize=10):
+        """
+        Writes a file on given PVC
+        Args:
+            pvc_obj: PVC object to write a file on
+            filesize: size of file to write (in GB)
+
+        Returns:
+            Pod on this pvc on which the file was written
+        """
+        pod_obj = self.pod_factory(
+            interface=self.interface, pvc=pvc_obj, status=constants.STATUS_RUNNING
+        )
+
+        # filesize to be written is always 10 GB
+        file_size = f"{int(filesize * 1024)}M"
+
+        log.info(f"Starting IO on the POD {pod_obj.name}")
+        # Going to run only write IO
+        pod_obj.fillup_fs(size=file_size, fio_filename=f"{pod_obj.name}_file")
+
+        # Wait for fio to finish
+        fio_result = pod_obj.get_fio_results()
+        err_count = fio_result.get("jobs")[0].get("error")
+        assert (
+            err_count == 0
+        ), f"IO error on pod {pod_obj.name}. FIO result: {fio_result}"
+        log.info("IO on the PVC has finished")
+        return pod_obj
+
+    @pytest.mark.usefixtures(base_setup.__name__)
+    def test_multiple_pvc_deletion_measurement_performance(self, teardown_factory):
+        """
+        Measuring PVC deletion time of 120 PVCs in 180 seconds
+
+        Args:
+            teardown_factory: A fixture used when we want a new resource that was created during the tests
+                               to be removed in the teardown phase.
+        Returns:
+
+        """
+        number_of_pvcs = 120
+        pvc_size = "1Gi"
+        msg_prefix = f"Interface: {self.interface}, PVC size: {pvc_size}."
+
+        log.info(f"{msg_prefix} Start creating new 120 PVCs")
+
+        pvc_objs = helpers.create_multiple_pvcs(
+            sc_name=self.sc_obj.name,
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+            number_of_pvc=number_of_pvcs,
+            size=pvc_size,
+        )
```

Comment:
you are creating the PVC's one by one and not in bulk, for bulk creation you need to add the argument `burst=true` (or just `true` at the end).

Reply:
@Avilir I've changed it, will run now to test it.